app/src/main/res/values/bools.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-  ~ This Source Code Form is subject to the terms of the Mozilla Public
-  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
-  ~ file, You can obtain one at http://mozilla.org/MPL/2.0/.
-  -->
-
-<resources>
-    <bool name="setting_telemetry_default">false</bool>
```

Comment:
maybe we can use your Constant.kt class moving forward? this was a default created with the TelemetryStore, so it's not used anymore now that I hooked up the SettingStore to the telemetry... what do you think?

Reply:
sure thing. Maybe for now, let's make the `rxSharedPreferences.getBoolean(...)` take a constant with a name like `SETTING_DEFAULT_SEND_USAGE_DATA` or something similar just so that it's obvious what that variable is doing.